spec/actions/user_spec.cr

```diff
@@ -0,0 +1,119 @@
+require "../spec_helper"
+
+module DppmRestApi::Actions::User
+  struct UserAddResponse
+    include JSON::Serializable
+    getter name : String
+    getter group_ids : Set(Int32)
+  end
+
+  struct UserDeleteResponse
+    struct Status
+      include JSON::Serializable
+      getter status : String
+    end
+
+    include JSON::Serializable
+    getter data : Status
+  end
+
+  struct UserGetResponse
+    struct Data
+      include JSON::Serializable
+      getter users : Array(User)
+
+      struct User
+        include JSON::Serializable
+        getter name : String
+        @[JSON::Field(ignore: true)]
+        getter config_user : DppmRestApi::Config::User do
+          DppmRestApi.permissions_config.users.find do |user|
+            user.name == name
+          end || fail "no user found in #{DppmRestApi.permissions_config.users} with the name #{name}"
+        end
+
+        def which_should_be_in_groups(groups : Enumerable(Int32))
+          groups.each do |group|
+            unless config_user.group_ids.includes? group
+              fail "user #{config_user} is not a member of the group #{group}"
+            end
+          end
+        end
+      end
+    end
+
+    include JSON::Serializable
+    getter data : Data
+
+    def should_contain_user_named(name : String)
+      user = data.users.find { |u| u.name == name }
```

Comment:
```suggestion
      user = data.users.find &.name.== name
```

Reply:
Will this work? I thought it wouldn't work unless you use parens like `&.name.==(name)`. Regardless, I think the full block syntax is less confusing when calling _operators_ on block data.

Reply:
It works.
`u` doesn't provide any information. We already know this are users, because of `data.users`. I don't think it's confusing, the syntax is quite common.